### ISSUE TITLE
I/O inside loop

### ISSUE THREAD

**Issue by leonardpitzu** [jaruba/ha-samsungtv-tizen#59]
in HA (latest stable & dev) i keep seeing this warning in the logs:

`Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for samsungtv_tizen doing I/O at custom_components/samsungtv_tizen/upnp.py, line 28: response = requests.post("http://{host}:9197/upnp/control/{protocole}1".format(host=self.host, protocole=protocole), data=body, headers=headers, timeout=0.1)`

**Comment by prsantos** [comment]
In HA Core 0.109.0, keep seeing this warning in logs:
Log Details (WARNING)
Logger: homeassistant.util.async_
Source: util/async_.py:120
First occurred: 8:43:06 PM (290 occurrences)
Last logged: 9:22:16 PM

Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for samsungtv_tizen doing I/O at custom_components/samsungtv_tizen/upnp.py, line 28: response = requests.post("http://{host}:9197/upnp/control/{protocole}1".format(host=self.host, protocole=protocole), data=body, headers=headers, timeout=0.1)

**Comment by phoniclynx** [comment]
```
Logger: homeassistant.util.async_
Source: util/async_.py:120
First occurred: 9:39:29 PM (26 occurrences)
Last logged: 9:42:41 PM

Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for samsungtv_tizen doing I/O at custom_components/samsungtv_tizen/upnp.py, line 28: response = requests.post("http://{host}:9197/upnp/control/{protocole}1".format(host=self.host, protocole=protocole), data=body, headers=headers, timeout=0.1)
```

**Comment by OptimusGREEN** [comment]
Im also seeing this issue in my logs.

**Comment by poisondima** [comment]
In HA Core 0.109.2 also warning in logs.

**Comment by Trikap82** [comment]
same here 109.2

**Comment by sanderlv** [comment]
same here... 
![image](https://user-images.githubusercontent.com/16528916/80871994-71861700-8caf-11ea-81aa-3035a23839ec.png)

I had this for another plugin and was solved in 2 days there:
https://github.com/pippyn/Home-Assistant-Sensor-Afvalbeheer/issues/69

**Comment by deten** [comment]
```
2020-05-03 13:49:53 WARNING (MainThread) [homeassistant.util.async_] Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for samsungtv_tizen doing I/O at custom_components/samsungtv_tizen/upnp.py, line 28: response = requests.post("http://{host}:9197/upnp/control/{protocole}1".format(host=self.host, protocole=protocole), data=body, headers=headers, timeout=0.1)
2020-05-03 13:49:53 WARNING (MainThread) [homeassistant.util.async_] Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for samsungtv_tizen doing I/O at custom_components/samsungtv_tizen/upnp.py, line 28: response = requests.post("http://{host}:9197/upnp/control/{protocole}1".format(host=self.host, protocole=protocole), data=body, headers=headers, timeout=0.1)
2020-05-03 13:49:53 WARNING (MainThread) [homeassistant.helpers.entity] Updating state for media_player.samsung_tv_remote (<class 'custom_components.samsungtv_tizen.media_player.SamsungTVDevice'>) took 1.158 seconds. Please report it to the custom component author.
```

**Comment by deten** [comment]
Note that the KS8000 Samsung TV requires port 8001, not 8002.

**Comment by jaruba** [comment]
Sorry for being inactive lately. I've made an announcement [here](https://community.home-assistant.io/t/improving-samsung-tv-control/151922/230?u=jaruba).

**Comment by jaruba** [comment]
I've released v1.4.7 that should handle the I/O error, please test it and if it continues to give the error reopen this issue.
